GPU/Common/FramebufferManagerCommon.cpp

```diff
@@ -943,6 +980,7 @@ void FramebufferManagerCommon::UpdateFromMemory(u32 addr, int size, bool safe) {
 	addr &= 0x3FFFFFFF;
 	// TODO: Could go through all FBOs, but probably not important?
 	// TODO: Could also check for inner changes, but video is most important.
+	// TODO: This shouldn't care if it's a display framebuf or not, should work exactly the same.
```

Comment:
UpdateFromMemory() is only called with `safe == true` now since block transfer changes, so this is a pretty old check.  Probably can be removed.  I think it does even go through all FBOs, I think this comment was from when we didn't and only wrote forcibly to the display framebuf on video write.  It's a pretty old func...

GPU_INVALIDATE_SAFE is used for video and PGF invalidations where we know the content should definitely be uploaded and isn't a mismatch.  It should probably be folded into the block transfer logic somehow.  I think it was left because people wanted videos to work with block transfer disabled.

-[Unknown]

Reply:
Right, I'll remove the safe flag from this function at least for now.

Really want to delete the block transfer option. Maybe it's time.